### ISSUE TITLE
linux 16.04 shutdown directly when evaluation

### ISSUE THREAD

**Issue by xllau** [multimodallearning/pytorch-mask-rcnn#36]
when I run this code for evaluation, the system directly shutdown. I tried to debug it. When I monitor the GPU memory, I found it soaring when jump into the forward() function of class Classifier, then shutdown.  When I delete the conv1-bn1-relu-conv2-bn2-relu in forward(), just remain roi_align, the evaluation can run well. But I don't figure out what's wrong with class classifier? 
It's an urgent problem, SOS! Plz help! @lasseha

    def forward(self, x, rois):
        x = pyramid_roi_align([rois]+x, self.pool_size, self.image_shape)
        x =  @self.conv1(x)
        x = self.bn1(x)
        x = self.relu(x)
        x = self.conv2(x)
        x = self.bn2(x)
        x = self.relu(x)

        x = x.view(-1,1024)
        mrcnn_class_logits = self.linear_class(x)
        mrcnn_probs = self.softmax(mrcnn_class_logits)

        mrcnn_bbox = self.linear_bbox(x)
        mrcnn_bbox = mrcnn_bbox.view(mrcnn_bbox.size()[0], -1, 4)

        return [mrcnn_class_logits, mrcnn_probs, mrcnn_bbox]
